docs/_plugins/markers-renderers/leaflet-coloricon.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Leaflet.ColorIcon
+category: markers-renderers
+repo: https://github.com/shevekk/Leaflet.ColorIcon
+author: Maxence Martin (shevek)
+author-url: https://github.com/shevekk
+demo: http://dataexplorer.hd.free.fr/Leaflet.ColorIcon/examples/basic/
+compatible-v0:
+compatible-v1: true
+---
+
+Create colored icons, allows to generate a css filter on the icon
```

Comment:
Maybe you want to change the text to something like that:

> Overwrite color of the icon with CSS filters and make it possible to add same icon-image in different colors.

Reply:
Thank you for your suggestion, I have changed the description